tests/B2CWebAppCallsWebApi/TodoListService/Controllers/TodoListController.cs

```diff
@@ -8,11 +8,13 @@
 using System.Collections.Generic;
 using System.Linq;
 using TodoListService.Models;
+using Microsoft.Identity.Web.Resource;
 
 namespace TodoListService.Controllers
 {
     [Authorize]
     [Route("api/[controller]")]
+    [RequiredScope("read")]
```

Comment:
>[RequiredScope("read")] [](start = 4, length = 23)

this is great! we really need to update the public one and keep these dev apps updated.